internal/core/application/operator_service.go

```diff
@@ -904,3 +834,66 @@ func (o *operatorService) getAllUnspentsForAccount(
 	}
 	return utxos, nil
 }
+
+func (o *operatorService) observeAddressesForAccount(accountIndex int, list []AddressAndBlindingKey) {
+	for _, l := range list {
+		blindkey, _ := hex.DecodeString(l.BlindingKey)
+		o.blockchainListener.StartObserveAddress(accountIndex, l.Address, blindkey)
+		time.Sleep(200 * time.Millisecond)
```

Comment:
We bring accross this sleep time. We shoudl have it a constant in a single place

Reply:
I agree, but since this isn't something we previously agreed on I wanted to know your opinions, guys.